user-profile-style.css

```diff
@@ -0,0 +1,12 @@
+.profile-header {
+
+}
+
+.profile-header__picture {
```

Comment:
Is "__" part of the style guide for css class naming? This is the first time I am seeing this style of naming.

Reply:
+1 Is there a particular reason for the double underscore?

Reply:
I went by this style for naming, somehow it helped me to keep track of things better. But if I need to change it, I will.
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/